auth/src/api/handlers.rs

```diff
@@ -102,7 +105,7 @@ pub(crate) async fn convert_cookie(
         .get::<AccountTier>("account_tier")
         .ok_or(StatusCode::UNAUTHORIZED)?;
 
-    let claim = Claim::new(account_name, account_tier.into());
+    let claim = Claim::new(account_name, account_tier.into(), account_tier);
```

Comment:
We could have only two args here, subject and account tier, then we create the scopes from the tier in the body of the new method as well. That would not allow us to manually add single scopes for testing, which [we do currently](https://github.com/shuttle-hq/shuttle/pull/1382/files#diff-37cffdd9fbd9d2426e519e0d75b4bf560cc755d5a11ee15b8977a338747253c3), but I'm not sure how useful it is, since those tests only add the minimum required scopes (basic would work too).

Reply:
I'll leave it in for readability.